### PR TITLE
fix calculation token amount -1

### PR DESCRIPTION
**Pull request by Nantawat-Poothong**
personal remain holidays token should be filter out before calculation with available holidays.

### CODE REVIEW

**Comment by Nantawat-Poothong** [comment]
@Sarin-Udompanish I think this logic would help. 🙂

File: src/components/pages/Home.js [hunk 1]
```diff
@@ -121,7 +121,7 @@ function Home() {
   const availableHolidays = defaultHolidays.filter(
     (holiday) => holiday.date >= currentDate
   )
-  const tokenAmount = availableHolidays.length - holidays.length
+  const tokenAmount = availableHolidays.length - holidays.filter(holiday => holiday >= currentDate).length
```
Comment (Nantawat-Poothong) [inline src/components/pages/Home.js:124]: @Sarin-Udompanish I think this logic would help. 🙂

**Comment by dtanp-rft** [comment]
Ok to close to this PR?

**Comment by Sarin-Udompanish** [comment]
Fixed.

**Pull request closed without merging**